Lib/test/test_pydoc.py

```diff
@@ -1276,11 +1275,9 @@ def __set__(self, obj, cls):
         class X:
             attr = Descr()
 
-        text = pydoc.plain(pydoc.render_doc(X.attr))
         self.assertEqual(self._get_summary_lines(X.attr), "")
 
         X.attr.__doc__ = 'Custom descriptor'
-        text = pydoc.plain(pydoc.render_doc(X.attr))
```

Comment:
There is yet one occurrence of unneeded code below.

Reply:
Ready, thanks